Makefile

```diff
@@ -31,7 +31,7 @@ dev: checklist check explaintest gogenerate br_unit_test test_part_parser_dev ut
 # Install the check tools.
 check-setup:tools/bin/revive tools/bin/goword
 
-check: fmt check-parallel lint tidy testSuite check-static vet errdoc
+check: check-parallel lint tidy testSuite check-static errdoc
```

Comment:
I think the golangci-lint didn't enable the govet.

And do we need to remove `fmt:`

Reply:
```fmt``` is replaced by ```build/linter/gofmt``` and it will run in the build.
```govet``` is the linter in the golang internal. so it has been enabled in the https://github.com/pingcap/tidb/blob/master/build/BUILD.bazel#L87

Reply:
Ok, do you think we need to delete https://github.com/pingcap/tidb/blob/0cecfc85afe7fdcb8ac397f9b8e50ab01b6a7f93/Makefile#L36-L39

Reply:
I think some developers may be able to use it.